### PR TITLE
Added a method to merge only known properties from one map to another

### PR DESCRIPTION
**Pull request by hernando**

### CODE REVIEW

File: brayns/common/PropertyMap.h [hunk 1]
```diff
@@ -355,6 +363,9 @@ class PropertyMap
     const auto& getProperties() const { return _properties; }
     /** Merge this property map with properties from another. */
     void merge(const PropertyMap& input);
+    /** Take the values from another property map only for properties that
+        are known and compatible to this one. */
```
Comment (tribal-tec) [inline brayns/common/PropertyMap.h]: if a property is incompatible it throws. Shall be mentioned for this and for `merge`

Comment (hernando) [reply brayns/common/PropertyMap.h]: That happens in both cases.

File: brayns/common/PropertyMap.h [hunk 2]
```diff
@@ -353,9 +358,19 @@ class PropertyMap
...
+     *  incompatible types is found in both maps.
+     */
     void merge(const PropertyMap& input);
 
+    /** Take the values from another property map only for properties that
+     *  are known and compatible to this one.
+     *  @throw std::runtime error if a property with the same name but
+     *  incompatible types is found in both maps.
+     */
+    void take(const PropertyMap& input);
```
Comment (karjonas) [inline brayns/common/PropertyMap.h]: Maybe we should call these takeAll and takeExisting to make the difference between the two methods clearer. They only differ in an else clause anyway.

Comment (hernando) [reply brayns/common/PropertyMap.h]: I prefer updateExisting rather than takeExisting and leave merge as merge.

**Pull request merged**